src/cronjobs/userProfileJobs.js

```diff
@@ -5,7 +5,15 @@ const userhelper = require('../helpers/userhelper')();
 const userProfileScheduledJobs = function () {
   const assignBlueBadge = new CronJob(
     '0 0 * * 0',
-    userhelper.assignBlueBadgeforTimeNotMet,
+    userhelper.assignBlueBadgesForTimeNotMetOrSummaries,
+    null,
+    false,
+    'America/Los_Angeles',
```

Comment:
```suggestion
    'America/Los_Angeles',
```
Here as well, will this need to be altered?

Reply:
Yes, this is actually crucial, it's important that it is PST here, again Jae's decision not mine.